assets/src/stories-editor/components/higher-order/with-amp-story-settings.js

```diff
@@ -247,6 +247,11 @@ export default createHigherOrderComponent(
 			const animationDelay = parseInt( String( ampAnimationDelay ).replace( 'ms', '' ) );
 
 			const captionAttribute = isVideoBlock ? 'ampShowCaption' : 'ampShowImageCaption';
+			// A block has text content (in terms of resizable at least) if it's a "real" text block with content or
+			// any of the meta text blocks
+			const isMetaTextBlock = name.indexOf( 'amp/amp-story-post-' ) === 0;
+			const hasTextContent = ( isTextBlock && content.length > 0 ) || isMetaTextBlock;
```

Comment:
Perhaps creating a new constant for meta blocks could be a better option? And then change `BLOCKS_WITH_TEXT_SETTINGS` to
```
export const BLOCKS_WITH_TEXT_SETTINGS = [
	'amp/amp-story-text',
	...POST_META_BLOCKS,
];
```
instead. Thoughts?

Reply:
Works for me!